runtime/client/src/clientde_impl.cpp

```diff
@@ -1441,11 +1441,6 @@ LTRESULT CLTClient::RemoveObject(HLOCALOBJ hObj)
 		RETURN_ERROR(1, CLTClient::RemoveObject, LT_INVALIDPARAMS);
 	}
 
-	if (!g_pClientMgr)
```

Comment:
this is a guard against `g_pClientMgr` being a nullptr when objectdll.lto is unloaded after clientshell.so,
please keep this